src/v2/view-builder/views/idp/RedirectIdvView.js

```diff
@@ -1,6 +1,22 @@
-import { loc } from '@okta/courage';
+import { loc, View } from '@okta/courage';
 import { BaseIdPAuthenticatorBody, BaseIdpAuthenticatorView } from './BaseIdpAuthenticator';
 import { FORMS } from 'v2/ion/RemediationConstants';
+import hbs from '@okta/handlebars-inline-precompile';
+import AuthenticatorFooter from 'v2/view-builder/components/AuthenticatorFooter';
+
+const PrivacyTermsFooterView = View.extend({
+  tagName: 'p',
+  className: 'margin-top-30',
```

Comment:
I'd discourage you from using a `className` just to add `margin` here. I took a look and we don't follow that pattern anywhere in gen2 code and we also prefer more semantic class names if we _have_ to use them.

Could you recreate the spacing you're looking for by adding the CSS you need to either a new .scss file in `assets/sass/v2` or maybe the existing `_footer.scss` file?

Maybe something like replacing the `class="margin-btm-5"` with `class="redirect-idv-footer"` and then defining something like `.redirect-idv-footer {
  margin: 20px 0 5px;
}`
This would allow you to be more exact with your margins because it seems like `30px` is a bit larger than the mockups you shared

Reply:
`margin-btm-*` `margin-top-*` are commonly used in okta-ui so i have used the same :)

updated the code to use custom class.